x-pack/package.json

```diff
@@ -195,6 +195,7 @@
     "@elastic/maki": "6.3.0",
     "@elastic/node-crypto": "1.1.1",
     "@elastic/numeral": "2.4.0",
+    "@elastic/safer-lodash-set": "0.0.0",
```

Comment:
Do we actually need to duplicate this when it's also in the root `package.json` file?

Reply:
@legrego do you know if this is necessary?

Reply:
I don't _think_ this is necessary. I believe that `x-pack/package.json` is only needed for dependencies that are only required by x-pack plugins, and isn't already part of the root `package.json`

Reply:
FYI: I had to re-add it to not have ESLint complain that it was missing. I think it would technically work just fine, but ESLint doesn't like it